### DISCUSSION TITLE
Zoom UI/All Tabs

### DISCUSSION THREAD

**Discussion by mrossinek** [qutebrowser/qutebrowser#7396] [Q&A] [answered]
Hi everyone!

Is there some command, which allows me to do either/both of the following at runtime?

1. zoom the UI (font size of statusbar, tabbar, etc.)
2. zoom all tabs at the same time

**Comment by The-Compiler** [comment, marked as answer]
The `fonts.default_size` and `fonts.tabs.*` settings.

**Reply by mrossinek** [reply]
So are you suggesting that I make a key mapping which changes those settings at runtime?

I'm assuming the same can be done for `zoom.default` but that does not trigger a re-render once I switch to the tab, does it?

**Reply by The-Compiler** [reply]
Yup, e.g. via `config-cycle` maybe. Setting `zoom.default` should immediately apply the change.
